Week1/homework/10-compare-arrays.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+const arr1 = ["My", "name", "is", "Sila"];
+const arr2 = ["Hi", "!", "my", "name", "is", "also", "Sila"];
```

Comment:
I'm glad that you used `const` here instead of `let` or `var` because we are not re-assigning `arr1` or `arr2`